gui/src/2D/controls/image.ts

```diff
@@ -307,14 +309,16 @@ export class Image extends Control {
 
     /** @hidden */
     public _rotate90(n: number, preserveProperties: boolean = false): Image {
-        let canvas = document.createElement("canvas");
-
-        const context = canvas.getContext("2d")!;
         const width = this._domImage.width;
         const height = this._domImage.height;
 
-        canvas.width = height;
-        canvas.height = width;
+        const engine = Engine.LastCreatedEngine;

```

Comment:
This doesn't feel right. If all of our platform abstractions are on engine, then this class needs to depend on engine. Using `LastCreatedEngine` seems wrong.

Reply:
I agree, I will change that.

Reply:
Actually no, I don't see how to get an engine instance without `LastCreatedEngine`. Any idea?